apps/storefront-e2e/src/integration/search.cy.ts

```diff
@@ -52,24 +52,19 @@ describe('Search suite', () => {
 
       // clear sorting and check that it is default again
       searchPage.getProductSorting().clearSorting();
-      searchPage.waitForSearchRequest();
```

Comment:
Why we are removing this wait? 

Without it we might have false positives here